File: src/cs/comparison_operators.cs
class Program
{
	public static void Main()
	{
		var result = 1 == 2;
		System.Console.WriteLine("1 equals 2 is " + result);
		
		result = 1 != 2;
		System.Console.WriteLine("1 not equals 2 is " + result);
		
		result = 1 > 2;
		System.Console.WriteLine("1 larger than 2 is " + result);
		
		result = 1 >= 2;
		System.Console.WriteLine("1 larger than or equals 2 is " + result);
		
		result = 1 < 2;
		System.Console.WriteLine("1 less than 2 is " + result);
		
		result = 1 <= 2;
		System.Console.WriteLine("1 less than or equals 2 is " + result);
	}
}

File: src/ts/comparison_operators.ts
import * as process from "process";

let result = 1 == 2;
process.stdout.write("1 equals 2 is " + result);

result = 1 != 2;
process.stdout.write("1 not equals 2 is " + result);

result = 1 > 2;
process.stdout.write("1 larger than 2 is " + result + "\n");

result = 1 >= 2;
process.stdout.write("1 larger than or equals 2 is " + result + "\n");

result = 1 < 2;
process.stdout.write("1 less than 2 is " + result + "\n");

result = 1 <= 2;
process.stdout.write("1 less than or equals 2 is " + result + "\n");